sources/web/datalab/polymer/modules/ApiManager.ts

```diff
@@ -216,6 +216,25 @@ class ApiManager {
   }
 
   /**
+   * Copy an item from source to destination. Item name collisions at the destination
+   * are handled by Jupyter.
+   * @param fromDir source path to copy from
+   * @param toPath destination path to copy to
+   */
+  static copyItem(fromPath: string, toPath: string) {
+    toPath = ApiManager.contentApiUrl + '/' + toPath;
+    const xhrOptions: XhrOptions = {
+      method: 'POST',
```

Comment:
The Jupyter backend expects `POST` to be used for copy. See [here](https://github.com/jupyter/notebook/blob/master/notebook/services/contents/handlers.py#L182) and [here](https://github.com/jupyter/notebook/blob/master/notebook/services/contents/handlers.py#L224).